src/main/java/pl/put/poznan/transformer/rest/ScenarioLinesWithoutActorInFirstWordController.java

```diff
@@ -0,0 +1,51 @@
+package pl.put.poznan.transformer.rest;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import pl.put.poznan.transformer.logic.domain.dto.ScenarioDTO;
+import pl.put.poznan.transformer.logic.service.ScenarioLinesWithoutActorInFirstWordService;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+public class ScenarioLinesWithoutActorInFirstWordController {
+    private final ScenarioLinesWithoutActorInFirstWordService scenarioLinesWithoutActorInFirstWordService;
+
+    public ScenarioLinesWithoutActorInFirstWordController(ScenarioLinesWithoutActorInFirstWordService scenarioLinesWithoutActorInFirstWordService) {
+        this.scenarioLinesWithoutActorInFirstWordService = scenarioLinesWithoutActorInFirstWordService;
+    }
+
+    /**
+     * Method: GET<br>
+     * Endpoint: /api/scenario/steps/count<br>
+     * @param scenarioDTO Object representing scenario created from parsed JSON body
+     * @return Object representing below JSON with example values:
+     *      <pre>
+     *      {
+     *          "lines": [
+     *              "0" :"SOMETHING ACTOR",
```

Comment:
It's not a valid JSON. Arrays can't have keys, only values. Furthermore if you give examples then it should be meaningful for the reader, make it look more like an actual step instruction(like in xls file example).